File: migration/pset1/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (1 > n || n > 8);
    
    for (int i = 0; i < n; i++)
    {
        for (int j = n - 1 ; j > i; j--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: migration/pset6/mario.py
from cs50 import get_int
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")
n = height
for i in range(height):
    for j in range(n-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print("  ", end = "")
    for j in range(i+1):
        print("#", end = "")
    print("")
    n = n - 1